libfreerdp/core/nego.h

```diff
@@ -92,6 +92,34 @@ enum RDP_NEG_MSG
 #define REDIRECTED_AUTHENTICATION_MODE_REQUIRED		0x02
 #define CORRELATION_INFO_PRESENT				0x08
 
+struct rdp_nego
```

Comment:
would prefer it if this struct stays opaque and you implrement access to the routing token via getter/setter